lava-v2-jobs-from-api.py

```diff
@@ -15,22 +33,16 @@
 from lib.utils import write_file
 import requests
 import urlparse
+import urllib
+import time
 from jinja2 import Environment, FileSystemLoader
 
 
 LEGACY_X86_PLATFORMS = ['x86', 'x86-kvm']
-LEGACY_ARM64_PLATFORMS = ['qemu-aarch64-legacy']
-INITRD_URLS = {'arm64': 'http://storage.kernelci.org/images/rootfs/buildroot/arm64/rootfs.cpio.gz',
-               'arm64be': 'http://storage.kernelci.org/images/rootfs/buildroot/arm64be/rootfs.cpio.gz',
-               'armeb': 'http://storage.kernelci.org/images/rootfs/buildroot/armeb/rootfs.cpio.gz',
-               'armel': 'http://storage.kernelci.org/images/rootfs/buildroot/armel/rootfs.cpio.gz',
-               'x86': 'http://storage.kernelci.org/images/rootfs/buildroot/x86/rootfs.cpio.gz'}
-NFSROOTFS_URLS = {'arm64': 'http://storage.kernelci.org/images/rootfs/buildroot/arm64/rootfs.tar.xz',
-               'arm64be': 'http://storage.kernelci.org/images/rootfs/buildroot/arm64be/rootfs.tar.xz',
-               'armeb': 'http://storage.kernelci.org/images/rootfs/buildroot/armeb/rootfs.tar.xz',
-               'armel': 'http://storage.kernelci.org/images/rootfs/buildroot/armel/rootfs.tar.xz',
-               'x86': 'http://storage.kernelci.org/images/rootfs/buildroot/x86/rootfs.tar.xz'}
-
+ARCHS = ['arm64', 'arm64be', 'armeb', 'armel', 'x86']
+ROOTFS_URL = 'http://storage.kernelci.org/images/rootfs'
+INITRD_URL = '/'.join([ROOTFS_URL, 'buildroot/{}/rootfs.cpio.gz'])
```

Comment:
I found the original list quite hard to manage, for example if you want to use binaries from a different place you then have to edit each line.  It's easy to get a typo wrong in a list like that.

If it's only me then I can drop this change, let me know.